<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>891</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Таблица товаров склада</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../build-pr_lab4_task2-Desktop_Qt_5_11_2_MinGW_32bit-Debug/icons/warehouse.png</normaloff>../build-pr_lab4_task2-Desktop_Qt_5_11_2_MinGW_32bit-Debug/icons/warehouse.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="table">
      <property name="autoScroll">
       <bool>false</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="verticalScrollMode">
       <enum>QAbstractItemView::ScrollPerItem</enum>
      </property>
      <property name="showGrid">
       <bool>true</bool>
      </property>
      <property name="gridStyle">
       <enum>Qt::SolidLine</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="panel" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QPushButton" name="addButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>310</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить товар</string>
       </property>
      </widget>
      <widget class="QLabel" name="nameLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Наименование:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="nameEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="stockLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>В наличии:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="buyerLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Покупатель:</string>
       </property>
      </widget>
      <widget class="QLabel" name="neededLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Необходимо:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="buyerEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="stockBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
      <widget class="QLabel" name="dateLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Дата получения:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="neededBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
      <widget class="QLabel" name="priceLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>260</y>
         <width>131</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Цена:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="priceBox">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>280</y>
         <width>231</width>
         <height>22</height>
        </rect>
       </property>
       <property name="maximum">
        <number>100000</number>
       </property>
      </widget>
      <widget class="QLineEdit" name="buyersFilterEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>520</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string comment="Покупатель" extracomment="Покупатель"/>
       </property>
      </widget>
      <widget class="QLabel" name="toaddFilterLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>450</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Показать товары, которые нужно пополнить:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="buyersFilterButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>520</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Показать</string>
       </property>
      </widget>
      <widget class="QLabel" name="buyersFilterLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>500</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Товары, заказанные покупетелем:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="toaddFilterButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>470</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Показать</string>
       </property>
      </widget>
      <widget class="QLabel" name="noneFilterLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>400</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Показать все товары:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="noneFilterButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>420</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Показать</string>
       </property>
      </widget>
      <widget class="QLabel" name="ungottenFilterLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>550</y>
         <width>241</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Показать покупателей, не получивших товар:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ungottenFilterButton">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>570</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Показать</string>
       </property>
      </widget>
      <widget class="QLabel" name="expensiveFilterLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>600</y>
         <width>231</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Самый дорогой заказ:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="expensiveFilterButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>620</y>
         <width>231</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Показать</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="ungottenFilterEdit">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>570</y>
         <width>111</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="placeholderText">
        <string comment="Дата" extracomment="Дата"/>
       </property>
      </widget>
      <widget class="QLabel" name="noneFilterLabel_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>171</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>ФОРМАТ ДАТЫ: dd/MM/yyyy</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>891</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="saveAction"/>
    <addaction name="separator"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>О программе</string>
    </property>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="openAction"/>
   <addaction name="saveAction"/>
   <addaction name="exitAction"/>
   <addaction name="separator"/>
   <addaction name="aboutAction"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="aboutAction">
   <property name="icon">
    <iconset>
     <normaloff>../build-pr_lab4_task2-Desktop_Qt_5_11_2_MinGW_32bit-Debug/icons/about.png</normaloff>../build-pr_lab4_task2-Desktop_Qt_5_11_2_MinGW_32bit-Debug/icons/about.png</iconset>
   </property>
   <property name="text">
    <string>Справка</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="openAction">
   <property name="icon">
    <iconset>
     <normaloff>../build-pr_lab4_task2-Desktop_Qt_5_11_2_MinGW_32bit-Debug/icons/open.png</normaloff>../build-pr_lab4_task2-Desktop_Qt_5_11_2_MinGW_32bit-Debug/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Открыть файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveAction">
   <property name="icon">
    <iconset>
     <normaloff>../build-pr_lab4_task2-Desktop_Qt_5_11_2_MinGW_32bit-Debug/icons/save.png</normaloff>../build-pr_lab4_task2-Desktop_Qt_5_11_2_MinGW_32bit-Debug/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Сохранить как...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset>
     <normaloff>../build-pr_lab4_task2-Desktop_Qt_5_11_2_MinGW_32bit-Debug/icons/exit.png</normaloff>../build-pr_lab4_task2-Desktop_Qt_5_11_2_MinGW_32bit-Debug/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
